tools/bash/runfiles/runfiles.bash

```diff
@@ -70,6 +70,7 @@
 #       # --- begin runfiles.bash initialization v3 ---
 #       # Copy-pasted from the Bazel Bash runfiles library v3.
 #       set -uo pipefail; set +e; f=bazel_tools/tools/bash/runfiles/runfiles.bash
+#       # shellcheck disable=SC1090
```

Comment:
The docs suggest a different workaround, should we use that instead?
https://www.shellcheck.net/wiki/SC1090

Reply:
I tried that but couldn't figure it out how to get it work because the file is part of runfile so depending on how shellcheck is executed, it may not be able to locate it. Having `# shellcheck source=/dev/null` is pretty much the same as ignoring.